src/test/java/seedu/address/logic/commands/EditListingCommandTest.java

```diff
@@ -63,21 +63,6 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         assertCommandSuccess(editListingCommand, model, expectedMessage, expectedModel);
     }
 
-    @Test
```

Comment:
Why did you remove this test?

Reply:
Yes Sean Kevin Dias, maybe enlighten us a little?